src/app/content/practiceQuestions/components/PracticeQuestionsPopup.spec.tsx

```diff
@@ -105,14 +118,13 @@ describe('PracticeQuestions', () => {
       closeButton.props.onClick();
     });
 
-    expect(dispatch).toHaveBeenCalledWith(closePracticeQuestions());
     expect(track).toHaveBeenCalled();
   });
 
-  it('closes popup on esc and tracks analytics', async() => {
+  it('tracks analytics when clicking esc', async() => {
```

Comment:
would be nice to check if `push` action  without modal parameter was called in these tests